packages/types/surveys.ts

```diff
@@ -511,57 +493,50 @@ export const ZSurvey = z.object({
   languages: z.array(ZSurveyLanguage),
 });
 
-export const ZSurveyWithRefinements = ZSurvey.refine((survey) => !surveyHasBothTriggers(survey), {
-  message: "Survey cannot have both triggers and inlineTriggers",
+export const ZSurveyWithRefinements = ZSurvey.extend({
```

Comment:
I think we can safely remove the `ZSurveyWithRefinements` schema from here, and just use the updated `ZSurvey` schema 😊